ddp-workspace/projects/ddp-pancan/src/styles/activities.scss

```diff
@@ -473,4 +473,142 @@ select:disabled {
     }
 }
 
-
+@media (min-width: 768px) {
+    [class*="composite-answer"][class*="GRID"] {
+        .ddp-answer-container {
+            margin-bottom: 2.5rem;
+
+            .ddp-answer-field {
+                display: block;
+
+                ddp-activity-picklist-answer {
+                    display: flex;
+
```

Comment:
#minor not sure we need to have empty lines between styles for the same selector, same for other places

Reply:
I just try to group css properties and separate them with empty lines.